.github/workflows/release.yml

```diff
@@ -86,11 +91,16 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
       
+      - name: Login to DigitalOcean Container Registry
+        run: doctl registry login --expiry-seconds 600
+      
       - name: Configure Docker image metadata
         id: meta
         uses: docker/metadata-action@v4
         with:
-          images: ghcr.io/${{ github.repository}}
+          images: |
+            ghcr.io/${{ github.repository}}
+            registry.digitalocean.com/${{ github.repository }}
```

Comment:
Unsure if build push action will successfully publish to both GCHR and DOCR. I know that GCHR is built-in, but DOCR pushes as far as I understand can only be done using DOCTL but maybe I'm wrong.

Reply:
This is just for metadata action and we have a separate docker login step with doctl. 
We use the same approach on [bdjuno](https://github.com/cheqd/bdjuno/blob/chains/cheqd/testnet/.github/workflows/release.yml#L48) and it works fine for now.

I saw on docker-metadata action documentation that they support multiple images so I think this should work.

Reply:
Let me rephrase the question and actually should have tagged a different section: to publish, if you use `build-push-action`, I think that will only be able to publish to support registry like GCHR. And you need to do a doctl push separately. Correct?

Reply:
You can push to DOCR with regular `docker push`, you just need separate docker login with doctl.

Tbh, I didn't test it yet with `build-push-action` but if they use the same underlying engine it should work.

I will make a separate step for DOCR anyway, just in case they don't support it.